src/starbucksDunkin.java

```diff
@@ -2,12 +2,21 @@
 public class starbucksDunkin {
 
 	public static void main(String[] args) {
-		Scanner sc = new Scanner(System.in);
 		System.out.println("Hi! What is your name?");
+		Scanner sc = new Scanner(System.in);
 		String name = sc.nextLine();
-		System.out.println("Nice to meet you.");
+		System.out.println("Nice to meet you.");		
 		System.out.println("I have a question for you, " + name + ".");
-		
+		System.out.println("How old are you?");
+
+		Scanner old = new Scanner(System.in);
+		int age = old.nextInt();
+		if (age < 18) {
```

Comment:
I liked this "age check" feature and thought using an int variable in an if statement was the best way to implement it.